packages/web-console/src/scenes/Editor/Monaco/index.tsx

```diff
@@ -203,9 +203,18 @@ const MonacoEditor = () => {
 
     // Insert query, if one is found in the URL
     const params = new URLSearchParams(window.location.search)
-    const query = params.get("query")
+    // Support multi-line queries (URL encoded)
+    const query = params.get("query")?.replaceAll("%0A", "\n")
```

Comment:
@insmac why was this necessary? `URLSearchParams.prototype.get` should already return decoded string. Maybe it didn't work for you on some browser?

Reply:
we double checked, it's not needed